app/views/transactions/index.html.erb

```diff
@@ -8,9 +8,6 @@
 <div id="transactions">
   <% @transactions.each do |transaction| %>
       <%= render transaction %>
-      <p>
-        <%= link_to "Show this transaction", transaction %>
-      </p>
   <% end %>
 </div>
 
```

Comment:
- [ ] Nice worko so far 👏  the login of the webApp is well implemented, however thr requirement is clear about the design of the project only the design of the template should be follow. Kindly make sure you follow the [design guidlines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) by using:
- Colors
- Typography: font face, size and weight.
- Layout: composition and space between elements.

Check the defference design pages

Your design | the template
---|---
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/606cb65f-b692-44ae-b6ae-54702f807f22"> | <img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/7a4705ce-6fe7-4b02-aaa6-107423d33da6">
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/9982a046-3120-4db2-ad78-0c00ef7a83cc"> | <img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/7dd31eb1-ffe8-426e-9957-de1a015c603a">  
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/308d70ff-5cf3-4f92-9a92-c98909ba98ca"> | <img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/c14efa79-37b0-46e0-8d77-f0abe97aedb5">
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/aa30615e-3fff-48b6-9975-db9e56930e35"> | <img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/db71d8de-556e-4627-b1b7-5d2de19da2c9">
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/d6a44e36-e826-48b2-9cc8-b7c130dfa4a6">
<img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/aa30615e-3fff-48b6-9975-db9e56930e35">  <img width="228" alt="image" src="https://github.com/V0pe/Payday/assets/65068771/e3da4037-1909-4685-a227-580dc006e3f4">

Reply:
@andersonlebon  Yes, You are right. I have adjusted the design. 
![image](https://github.com/V0pe/Payday/assets/61356487/a3d0e54f-84df-4bb0-b213-ba9b3cf56983)
![image](https://github.com/V0pe/Payday/assets/61356487/24937dae-d70f-41c3-9fda-43a66541dc51)
![image](https://github.com/V0pe/Payday/assets/61356487/01af776f-6840-4cd9-9f0d-98b396ee6184)
![image](https://github.com/V0pe/Payday/assets/61356487/f5d2788f-a5a8-4b26-8816-d16539cfb53b)
![image](https://github.com/V0pe/Payday/assets/61356487/880284cf-c0e1-487e-9021-6961469e3630)